bin/PrintGCStats

```diff
@@ -805,6 +807,19 @@ function writePlotData(tstamp, value, widx) {
 }
 
 function recordStats(idx, value) {
+  rs_tstamp = -1;
+  if (available_plot_count != 0 && (idx in plot_idx || (totgc_idx in plot_idx && idx < totgc_idx))) {
+    # Plotting enabled, need to find timestamp on line
+    if (datestamps == 1) {
+      rs_tstamp = getDateTimeStamp();
+    } else {
+      rs_tstamp = getTimeStamp();
+    }
+  }
+  recordStatsTimestamp(idx, value, rs_tstamp);
```

Comment:
Hi.

To be able to parse the following lines 
```
9.563: [GC (Allocation Failure) 9.563: [ParNew (promotion failed): 3071K->3071K(3072K), 0.0169675 secs]9.580: [CMS2015-01-11T12:06:30.899+0100: 9.597: [CMS-concurrent-mark: 0.077/0.440 secs]
(concurrent mode failure): 511643K->511999K(512000K), 0.4506954 secs] 513856K->512001K(515072K), [Metaspace: 2428K->2428K(1056768K)], 0.4681062 secs]
```
and
```
10.764: [Full GC (Allocation Failure) 10.764: [CMS2015-01-11T12:06:32.071+0100: 10.769: [CMS-concurrent-mark: 0.137/0.137 secs]
(concurrent mode failure): 511999K->511999K(512000K), 0.3167690 secs] 742462K->640024K(742464K), [Metaspace: 2428K->2428K(1056768K)], 0.3168884 secs]
```
and record the gen1t, the second line (where the gen1 data is), needs to use the timestamp from the previous line.

Therefore I introduced recordStatsTimestamp, which takes the timestamp as input parameter.
The old method recordStats finds the timestamp from the line it receives, and then uses recordStatsTimestamp to actually record the data.
For the concurrent mode failure, I do like this
```
# Record stats with timestamp from the previous line
recordStatsTimestamp(gen1t_idx, secs, rs_tstamp);
```

Does this seem OK, or should I do it in a different manner ?

Alf